libpod/container_top_linux.go

```diff
@@ -51,7 +51,17 @@ func (c *Container) Top(descriptors []string) ([]string, error) {
 		return nil, psgoErr
 	}
 
-	output, err = c.execPS(descriptors)
+	// Note that the descriptors to ps(1) must be split on ` ` (see #12452).
+	psDescriptors := []string{}
+	for _, d := range descriptors {
+		for _, s := range strings.Split(d, " ") {
```

Comment:
Do we need to respect shell quoting?

Reply:
Definitely more robust, let me repush in a jiffy.